src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -133,27 +135,27 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
               />
             </FormControl>
           </Grid>
-          <Grid item xs={12}>
-            <FormControl sx={{ minWidth: 325, width: '37%' }}>
-              <FormLabel>Project Summary</FormLabel>
-              <ReactHookTextField
-                name="summary"
-                control={control}
-                placeholder="Enter summary..."
-                errorMessage={errors.summary}
-                multiline
-                rows={5}
-              />
-            </FormControl>
-          </Grid>
         </Grid>
-        <Box display="flex" gap={2} sx={{ mt: 2 }}>
-          <SubmitButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
-            Create
-          </SubmitButton>
+        <Grid item xs={12} md={3}>
```

Comment:
this grid item is not in a container you accidently ended the grid container in the line above

Reply:
also you should delete the md prop because we always want this to take up the fullwidth of the grid